media.module

```diff
@@ -22,3 +23,38 @@ function media_help($route_name, RouteMatchInterface $route_match) {
     default:
   }
 }
+
+/**
+ * Implements hook_form_alter().
+ */
+function media_form_alter(&$form, FormStateInterface &$form_state) {
+  if (isset($form['#form_id'])) {
+    if ($form['#form_id'] == 'entity_browser_media_library_form') {
+      // Attach our library.
+      $form['#attached']['library'][] = 'media/view';
```

Comment:
I was working on the user view. I have attached it now. I'll update the cs and js files soon